x-pack/plugins/apm/server/lib/services/get_service_error_groups/get_service_error_group_primary_statistics.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { orderBy } from 'lodash';
+import {
+  ERROR_EXC_MESSAGE,
+  ERROR_GROUP_ID,
+  ERROR_LOG_MESSAGE,
+  SERVICE_NAME,
+  TRANSACTION_TYPE,
+} from '../../../../common/elasticsearch_fieldnames';
+import { NOT_AVAILABLE_LABEL } from '../../../../common/i18n';
+import { ProcessorEvent } from '../../../../common/processor_event';
+import { rangeFilter } from '../../../../common/utils/range_filter';
+import { withApmSpan } from '../../../utils/with_apm_span';
+import { getErrorName } from '../../helpers/get_error_name';
+import { Setup, SetupTimeRange } from '../../helpers/setup_request';
+
+export function getServiceErrorGroupPrimaryStatistics({
+  serviceName,
+  setup,
+  transactionType,
+}: {
+  serviceName: string;
+  setup: Setup & SetupTimeRange;
+  transactionType: string;
+}) {
+  return withApmSpan('get_service_error_group_primary_statistics', async () => {
+    const { apmEventClient, start, end, esFilter } = setup;
+
+    const response = await apmEventClient.search({
+      apm: {
+        events: [ProcessorEvent.error],
+      },
+      body: {
+        size: 0,
+        query: {
+          bool: {
+            filter: [
+              { term: { [SERVICE_NAME]: serviceName } },
+              { term: { [TRANSACTION_TYPE]: transactionType } },
+              { range: rangeFilter(start, end) },
+              ...esFilter,
+            ],
+          },
+        },
+        aggs: {
+          error_groups: {
+            terms: {
+              field: ERROR_GROUP_ID,
+              size: 500,
+              order: {
+                _count: 'desc',
+              },
+            },
+            aggs: {
+              sample: {
+                top_hits: {
+                  size: 1,
+                  _source: [ERROR_LOG_MESSAGE, ERROR_EXC_MESSAGE, '@timestamp'],
+                  sort: {
+                    '@timestamp': 'desc',
+                  },
+                },
+              },
+            },
+          },
+        },
+      },
+    });
+
+    const errorGroups =
+      response.aggregations?.error_groups.buckets.map((bucket) => ({
+        group_id: bucket.key as string,
+        name:
+          getErrorName(bucket.sample.hits.hits[0]._source) ??
+          NOT_AVAILABLE_LABEL,
+        last_seen: new Date(
+          bucket.sample.hits.hits[0]?._source['@timestamp']
+        ).getTime(),
+        occurrences: {
+          value: bucket.doc_count,
+        },
```

Comment:
As mentioned, let's removes the nesting here.

```suggestion
        occurrencesCount: bucket.doc_count
```